### PR TITLE
[TEVA-1770] Add apply_through_teaching_vacancies

### PR DESCRIPTION
**Pull request by cesidio**
## Jira ticket URL
https://dfedigital.atlassian.net/browse/TEVA-1770

## Screenshots of UI changes:
Screenshot 2021-02-08 at 17 42 56
Screenshot 2021-02-08 at 17 43 14
Screenshot 2021-02-08 at 17 43 57
Screenshot 2021-02-08 at 17 44 13
Screenshot 2021-02-09 at 11 19 11

### CODE REVIEW

File: app/views/publishers/vacancies/build/applying_for_the_job.html.slim [hunk 1]
```diff
@@ -14,6 +14,14 @@
 
       = render "publishers/vacancies/vacancy_form_partials/hidden_state_input", f: f
 
+      - if JobseekerApplicationsFeature.enabled?
+        = f.govuk_radio_buttons_fieldset :apply_through_teaching_vacancies do
+          = f.govuk_radio_button :apply_through_teaching_vacancies, :yes, link_errors: true do
+            = f.govuk_text_area :personal_statement, label: { size: "s" }
```
Comment (cpjmcquillan) [inline app/views/publishers/vacancies/build/applying_for_the_job.html.slim:20]: Can we call this `personal_statement_guidance` maybe?

File: config/locales/jobs.yml [hunk 2]
```diff
@@ -228,15 +228,16 @@ en:
...
 
     applying_for_the_job: Applying for the job
+    apply: More on how to apply
+    application_link: Link to online application
+    apply_through_teaching_vacancies: Apply through Teaching Vacancies
     contact_email: Contact email
     contact_email_subject: '%{job} job enquiry'
     contact_email_body: Regarding the job at %{url}
     contact_number: Contact phone number
+    personal_statement: Personal statment instructions
```
Comment (cpjmcquillan) [inline config/locales/jobs.yml:238]: typo

File: config/locales/forms.yml [hunk 3]
```diff
@@ -359,6 +365,8 @@ en:
       personal_details_form:
         address_html: Your address (<span class='text-red'>Required</span>)
 
+      publishers_job_listing_applying_for_the_job_form:
+        apply_through_teaching_vacancies_html: How would you like candidates to apply (<span class='text-red'>Required</span>)
```
Comment (cpjmcquillan) [inline config/locales/forms.yml:369]: missing `?`

**Review by cpjmcquillan — APPROVED** [review]
Just a couple of minor comments.

Do we also want to add the logic for the `Apply for this job` buttons based on the answer to this question?

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-2811.london.cloudapps.digital>

**Comment by cesidio** [comment]
> Just a couple of minor comments.
> 
> Do we also want to add the logic for the `Apply for this job` buttons based on the answer to this question?

There is more to do on the jobseeker page, like displaying the extra field that we added, maybe it makes sense to have another ticket that covers all of that

**Pull request merged**